data/mt/retired/Willis-Curdy-d6a3a052-3fb3-4dce-9ce0-074852fb2612.yml

```diff
@@ -7,7 +7,8 @@ email: Willis.Curdy@mtleg.gov
 party:
 - name: Democratic
 roles:
-- type: lower
+- end_date: '2022-12-31'
+  type: lower
   jurisdiction: ocd-jurisdiction/country:us/state:mt/government
```

Comment:
@bfossen-ce this is a duplicate from earlier in the pull. They served in the house from 2015 until 01/02/2023. On that date they start in the senate